crates/cargo-test-support/src/registry.rs

```diff
@@ -118,7 +149,7 @@ impl RegistryBuilder {
     pub fn new() -> RegistryBuilder {
         RegistryBuilder {
             alternative: None,
-            token: None,
+            token: Token::Bearer("api-token".to_string()),
```

Comment:
Why this change? Previously a value of `None` would be changed to the default of `"{prefix}sekrit"`.

Reply:
Good catch. Will be fixed in the next push.